src/index.html

```diff
@@ -1,13 +1,75 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div
+    class="card"
+    data-qa="card"
+    >
+      <a href="#">
+        <img
+        src="images/imac.jpeg"
+        alt="photo"
+        class="card__photo"
+        >
+      </a>
+      <div class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
```

Comment:
здесь лучше записать в несколько строк
```suggestion
      <div class="card__name">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </div>
```